sdk/identity/azure-identity-v2/src/main/java/com/azure/identity/v2/implementation/models/HttpPipelineOptions.java

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.v2.implementation.models;
+
+import io.clientcore.core.http.client.HttpClient;
+import io.clientcore.core.http.pipeline.HttpInstrumentationOptions;
+import io.clientcore.core.http.pipeline.HttpPipeline;
+import io.clientcore.core.http.pipeline.HttpPipelinePolicy;
+import io.clientcore.core.http.pipeline.HttpRetryOptions;
+import io.clientcore.core.http.pipeline.HttpRedirectOptions;
+
+import java.util.List;
+
+/**
+ * Represents Http Pipeline Options used when constructing the Http Pipeline.
+ */
+public class HttpPipelineOptions implements Cloneable {
```

Comment:
This feels a little like a thing that should live in core and not here? I know there's been a ton of discussion about it that hasn't landed maybe..